hosted/json-schemas/rfq.schema.json

```diff
@@ -1,47 +1,51 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "$id": "https://tbdex.dev/rfq.schema.json",
-  "definitions": {
-    "SelectedPaymentMethod": {
-      "type": "object",
-      "additionalProperties": false,
-      "properties": {
-        "kind": {
-          "type": "string",
-          "description": "Type of payment method e.g. BTC_ADDRESS, DEBIT_CARD, MOMO_MPESA"
-        },
-        "paymentDetails": {
-          "type": "object",
-          "description": "An object containing the properties defined in the respective Offering's requiredPaymentDetails json schema"
-        }
-      },
-      "required": ["kind"]
-    }
-  },
   "type": "object",
   "properties": {
     "additionalProperties": false,
     "offeringId": {
       "type": "string",
       "description": "Offering which Alice would like to get a quote for"
     },
-    "payinAmount": {
-      "$ref": "definitions.json#/definitions/decimalString",
-      "description": "Amount of quote currency you want to spend in order to receive base currency"
-    },
     "claims": {
       "type": "array",
       "items": {
         "type": "string"
       },
       "description": "Presentation Submission that fulfills the requirements included in the respective Offering"
     },
-    "payinMethod": {
-      "$ref": "#/definitions/SelectedPaymentMethod"
+    "payin": {
+      "type": "object",
+      "properties": {
+        "amount": {
+          "$ref": "definitions.json#/definitions/decimalString"
+        },
+        "kind": {
+          "type": "string",
+          "description": "Type of payment method e.g. BTC_ADDRESS, DEBIT_CARD, MOMO_MPESA"
+        },
+        "paymentDetails": {
+          "type": "object",
+          "description": "An object containing the properties defined in the respective Offering's requiredPaymentDetails json schema"
```

Comment:
could a `$ref` be added here?

Reply:
what do you have in mind @decentralgabe ? this property's value is an object that meets the requirements set by the offering's `requiredPaymentDetails`